### PR TITLE
Use data-driven-motion to animate demo.

### PR DESCRIPTION
**Pull request by tkh44**
I used https://github.com/tkh44/data-driven-motion to animate the demo because I thought it would look cool.

### CODE REVIEW

File: src/components/DataView/index.js [hunk 1]
```diff
@@ -1,95 +1,206 @@
...
+	nArray: {
+		rx: 5,
+		ry: 5,
+		marginLeft: 2,
+		marginBottom: 2
+	},
+	matrix: {}
+};
+
 const getComponent = (data, styles) => outputType => {
```
Comment (tkh44) [inline src/components/DataView/index.js:73]: This is the important stuff.

File: src/components/DataView/index.js [hunk 2]
```diff
@@ -1,95 +1,206 @@
...
+import { makeCoordinateArray as makeMatrixArray } from './Matrix';
+import { makeCoordinateArray as makeArrayArray } from './Array';
+import { makeCoordinateArray as makeNArrayArray } from './NArray';
+import Cell from './Cell';
+import { minDevice } from '../../lib/css';
+import { max, min } from 'd3-array';
+import { scaleLinear } from '@vx/scale/build/index';
 
-const borderRadius = "10px";
+const STIFF_SPRING = { stiffness: 1028, damping: 36 };
```
Comment (tkh44) [inline src/components/DataView/index.js]: You can change these numbers.

http://chenglou.github.io/react-motion/demos/demo5-spring-parameters-chooser/

**Comment by trainorpj** [comment]
This looks great! This is exactly the animation I wanted, but I had no idea how to implement it.

I can take a look at this later tonight. Thanks!

**Comment by tkh44** [comment]
Sorry about the formatting. I just ran eslint --fix and it created all these changes.

**Pull request merged**